pkg/operator/events/eventstesting/recorder_wrapper.go

```diff
@@ -11,6 +12,10 @@ type EventRecorder struct {
 	testingEventRecorder *TestingEventRecorder
 }
 
+func (e *EventRecorder) WithContext(ctx context.Context) events.Recorder {
+	return e
+}
+
```

Comment:
what is this?

Reply:
this is a "eventtesting" wrapper, I need to to meet the interface, but in testing we never use the context, so no need to store it.